src/pages/Login/Login.module.css

```diff
@@ -34,3 +34,40 @@
     justify-content: center;
 }
 
+
+.tooltipContainer {
+    position: relative;
+    display: inline-block;
```

Comment:
does `display: inline-block` affect anything? I tried to toggle it and it made no difference. Is there some specific case for it?